scripts/install_plugin_modules.js

```diff
@@ -68,6 +67,10 @@ function assertFolder (name, version) {
     fs.mkdirSync(folder())
   }
 
+  if (name && name.includes(path.sep)) {
```

Comment:
This is meant to support the `@grpc/proto-loader` dependency defined in the externals.json file. Without it you get an error since we'll be trying to "mkdir" a folder with a path separator.

Reply:
Thanks for clarifying, I forgot about the externals.